generators/python/src/fern_python/generators/sdk/client_generator/request_body_parameters/file_upload_request_body_parameters.py

```diff
@@ -25,13 +25,17 @@ def __init__(
     def get_parameters(self, names_to_deconflict: Optional[List[str]] = None) -> List[AST.NamedFunctionParameter]:
         parameters: List[AST.NamedFunctionParameter] = []
         for property in self._request.properties:
+            type_hint = self._get_property_type(property)
             parameters.append(
                 AST.NamedFunctionParameter(
                     name=self._get_property_name(property),
-                    type_hint=self._get_property_type(property),
+                    type_hint=type_hint,
                     docs=self._get_docs(property),
                     raw_type=self._get_raw_property_type(property),
                     raw_name=self._get_raw_property_name(property),
+                    initializer=AST.Expression(DEFAULT_BODY_PARAMETER_VALUE)
```

Comment:
If not specified, we actually default optional params to None, which is no good here